Formula/ford.rb

```diff
@@ -13,52 +13,63 @@ class Ford < Formula
     sha256 "0b513d22c17e06f4d7021d9f69a0254a33c24e5197b02224280e275af0599971" => :mavericks
   end
 
+  option "without-lxml", "Do not install lxml to improve the speed of search  database generation"
+
   depends_on "graphviz"
   depends_on :python if MacOS.version <= :snow_leopard
 
+  if build.with? "lxml"
+    resource "lxml" do
+      url "https://pypi.python.org/packages/11/1b/fe6904151b37a0d6da6e60c13583945f8ce3eae8ebd0ec763ce546358947/lxml-3.6.0.tar.gz"
+      sha256 "9c74ca28a7f0c30dca8872281b3c47705e21217c8bc63912d95c9e2a7cac6bdf"
+    end
+  end
```

Comment:
Derps, sorry about that. Will attend to it shortly.
On Sat, Jun 11, 2016 at 7:03 PM Martin Afanasjew <notifications@github.com>
wrote:

> In Formula/ford.rb
> <https://github.com/Homebrew/homebrew-core/pull/1578#discussion_r66713011>
> :
>
> >    depends_on "graphviz"
> >    depends_on :python if MacOS.version <= :snow_leopard
> >
> > +  if build.with? "lxml"
> > +    resource "lxml" do
> > +      url "https://pypi.python.org/packages/11/1b/fe6904151b37a0d6da6e60c13583945f8ce3eae8ebd0ec763ce546358947/lxml-3.6.0.tar.gz"
> > +      sha256 "9c74ca28a7f0c30dca8872281b3c47705e21217c8bc63912d95c9e2a7cac6bdf"
> > +    end
> > +  end
>
> Please remove the outer if. Resources should always be listed
> unconditionally, though they might not always be needed depending on the
> build flags (and how the install method handles them).
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Homebrew/homebrew-core/pull/1578/files/09c650c1619075cbf87875117258caf9dc28a211#r66713011>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAREPLbMqTDDubbjllJK48Ic63h1MWN7ks5qKz6igaJpZM4IqDrW>
> .
>